README.md

```diff
@@ -100,6 +100,50 @@ client.set_pool_info("autoscale",
     )
 ```
 
+**Configuration**
+An AzureClient object can be instantiated and initialized with pool, mounted containers and container registries using a configuration file. This is especially useful if the same pool will be used for running multiple batch jobs and experiments. Use the following example to create a configuration file:
+
+```

```

Comment:
it might be better to move this configuration to an example file that you can hyperlink to, so that we reduce the text on the readme. the readme is already getting long